compute_tools/src/config.rs

```diff
@@ -34,17 +34,23 @@ pub fn line_in_file(path: &Path, line: &str) -> Result<bool> {
 /// Create or completely rewrite configuration file specified by `path`
 pub fn write_postgres_conf(path: &Path, spec: &ComputeSpec) -> Result<()> {
     // File::create() destroys the file content if it exists.
-    let mut postgres_conf = File::create(path)?;
+    let mut file = File::create(path)?;
 
-    write_auto_managed_block(&mut postgres_conf, &spec.cluster.settings.as_pg_settings())?;
+    writeln!(file, "# Managed by compute_ctl: begin")?;
 
-    Ok(())
-}
+    writeln!(file, "{}", &spec.cluster.settings.as_pg_settings())?;
+
+    match spec.mode {
+        ComputeMode::Primary => {}
+        ComputeMode::Static(lsn) => {
+            write!(file, "hot_standby=on")?;
```

Comment:
I recall some discussion, but why do we set `hot_standby=on` if default is `on` already? As far as I see, we add `standby.signal`, which should be a real 'signal' for Postgres to be a replica, right?

Reply:
Before this PR, `compute_ctl` looked at the settings in the spec file, and if it contained `hot_standby=on`, it launched postgres as a read-only replica. It didn't make any different to Postgres, because `hot_standby` is on by default, but it was a signal to `compute_ctl`.

That was pretty hacky, which is why this PR is a good idea :-).

Reply:
Reading your comment again, I see that I didn't answer to your question. I explained why we set `hot_standby=on` _in the control plane_, when creating a read only compute, but that's not what you asked about.

You're right that we wouldn't need to set `hot_standby=on` here, because it's the default anyway. I think it's still nice to do set it explicitly, for documentation purposes if nothing else.